addon/model.js

```diff
@@ -102,41 +103,50 @@ const retrieveFromCurrentState = computed('_topModel.currentState', function(key
   return this._topModel._internalModel.currentState[key];
 }).readOnly();
 
-// global buffer for initial properties to work around
-//  a)  can't write to `this` before `super`
-//  b)  core_object writes properties before calling `init`; this means that no
-//      CP or setknownProperty can rely on any initialization
-let initProperites = Object.create(null);
+const INIT_KEYS = {
```

Comment:
Please don't use all caps for var names.

Kind of annoying that we can't configure eslint for this.  🤷‍♂️ 

`InitKeys` or `Init_Keys` is fine.